datavisapp/create_db.py

```diff
@@ -1,30 +1,39 @@
 import json
 import sqlite3
-from collections import OrderedDict
-
 import click as click
 import pandas as pd
 
+from collections import OrderedDict
+
 
-def make_table(db, table_name, col_types):
-    conn = sqlite3.connect(db)
-    cursor = conn.cursor()
+def make_table(table_name, col_types, db_name=':memory:'):
```

Comment:
How about changing the parameters of these databasing function to accept a connection, rather than a database? Then you can do a single connect() at the top level and use it for all the table-making and record-inserting and querying. And it would simplify the code a little bit.

Reply:
Nice - definitely a better approach!